build.properties

```diff
@@ -90,7 +90,7 @@ jfx.jdk.target.version=17
 # The jfx.gradle.version.min property defines the minimum version of gradle
 # that is supported. It must be <= jfx.gradle.version.
 jfx.gradle.version=7.6
-jfx.gradle.version.min=7.3
+jfx.gradle.version.min=7.6
```

Comment:
We only bump the minimum gradle version when we also bump the minimum JDK version or when we make changes in `build.gradle` that necessitate it. Since gradle 7.3 can still be used to build with JDK 17 (which is the minimum boot JDK version), there is no need to bump the minimum version of gradle this time.

Reply:
Thanks, Reverted the min version to 7.3.